RecoBTag/TensorFlow/python/pfMassIndependentDeepDoubleCvBJetTags_cfi.py

```diff
@@ -1,21 +1,12 @@
 import FWCore.ParameterSet.Config as cms
+from pfDeepDoubleBvLJetTags_cfi import pfDeepDoubleBvLJetTags
 
-pfMassIndependentDeepDoubleCvBJetTags = cms.EDProducer('DeepDoubleXTFJetTagsProducer',
-  src = cms.InputTag('pfDeepDoubleXTagInfos'),
-  input_names = cms.vstring(
-    'input_1',
-    'input_2',
-    'input_3'
-  ),
-  graph_path = cms.FileInPath('RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDCvB_mass_independent.pb'),
-  lp_names = cms.vstring('db_input_batchnorm/keras_learning_phase'),
-  output_names = cms.vstring('ID_pred/Softmax'),
-  flavor = cms.string('CvB'),
-  flav_table = cms.PSet(
-    probHbb = cms.vuint32(0),
-    probHcc = cms.vuint32(1)
-  ),
-  batch_eval = cms.bool(False),
-  nThreads = cms.uint32(1),
-  singleThreadPool = cms.string('no_threads')
+pfMassIndependentDeepDoubleCvBJetTags = pfDeepDoubleBvLJetTags.clone(
+    flavor = 'CvB',
+    graph_path = cms.FileInPath('RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDCvB_mass_independent.pb'),
```

Comment:
apparently, this should simply clone from already existing pfDeepDoubleCvBJetTags:
```pfMassIndependentDeepDoubleCvBJetTags = pfDeepDoubleCvBJetTags.clone(graph_path = 'RecoBTag/Combined/data/DeepDoubleX/94X/V01/DDCvB_mass_independent.pb')```

Also, in similar cases with few-lines definitions of modules in CMSSW we usually have an aggregate cff file used to define multiple modules.
E.g. ```pfMassIndependentDeepDoubleXJetTags_cff.py``` to contain BvL, CvB and CvL
This may be a simpler solution

Reply:
ok